### PR TITLE
Citation info added

### PR DESCRIPTION
**Pull request by massimoaria**
added information concerning the citation of the  openalexR paper in R Journal

(Yes, it is finally online!)

### CODE REVIEW

**Review by trangdata — APPROVED** [review]
Great news, Massimo! 🎉

File: inst/CITATION [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+         author = "Aria Massimo and Le Trang and Cuccurullo Corrado and Belfiore Alessandra and Choe June",
+         journal = "The R Journal",
+         year = "2024",
+         doi = "https://doi.org/10.32614/RJ-2023-089",
+         volume = "15",
+         issue = "4",
+         issn = "2073-4859",
+         pages = "167-180",
+         textVersion =
+           paste("Aria, M., Le, T., Cuccurullo, C., Belfiore, A. & Choe, J. (2014),",
```
Comment (yjunechoe) [inline inst/CITATION:14]: 2024

File: README.Rmd [hunk 2]
```diff
@@ -34,6 +34,12 @@ knitr::opts_chunk$set(
 
 -   `oa_random`: get random entity, *e.g.*, `oa_random("works")` gives a different work each time you run it
 
+## 📜 Citation
+
+If you use openalexR in research, please cite:
+
+  Aria, M., Le T., Cuccurullo, C., Belfiore, A. & Choe, J. (2014), *openalexR: An R-Tool for Collecting Bibliometric Data from OpenAlex*, **The R Journal**, 15(4), 167-180, DOI: https://doi.org/10.32614/RJ-2023-089.
```
Comment (yjunechoe) [inline README.Rmd:41]: 2024

File: README.md [hunk 3]
```diff
@@ -39,6 +39,15 @@ publishers, topics and concepts with 5 main functions:
 - `oa_random`: get random entity, *e.g.*, `oa_random("works")` gives a
   different work each time you run it
 
+## 📜 Citation
+
+If you use openalexR in research, please cite:
+
+Aria, M., Le T., Cuccurullo, C., Belfiore, A. & Choe, J. (2014),
```
Comment (yjunechoe) [inline README.md:46]: Upstream fix in .Rmd should resolve this

**Review by yjunechoe — COMMENTED** [review]
Found a small typo for year (2014 -> 2024) :)

Hurray!

File: README.Rmd (hunk 2 above)
Comment (trangdata) [reply README.Rmd:41]: Ah I completely missed this!

**Pull request merged**